src/modules/Home/components/GratitudeNote.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Container,
+  Flex,
+  ListItem,
+  Text,
+  UnorderedList,
+} from '@chakra-ui/react'
+
+const GratitudeNote = () => {
+  return (
+    <Container maxW="container.xl" w="100%" mb={55}>
+      <Flex
+        flexDirection="column"
+        align="center"
+        gap={5}
+        pt={9}
+        pb={69}
+        px={139}
+        bg="rgba(255, 255, 255, 0.7)"
+        boxShadow="0 4px 4px rgba(0,0,0,.25)"
+      >
+        <Text color="cyanBlue.800" fontSize={28} fontWeight={700}>
+          Dziękujemy za wybór Neptunes Sushi Delivery!
+        </Text>
+
+        <Text color="cyanBlue.800" fontSize={28} fontWeight={700}>
+          Nasze sushi zostały przygotowane według przepisu samego króla mórz
+          Neptuna, co oznacza, że zapewniamy ci:
+        </Text>
+
+        <UnorderedList>
```

Comment:
ListItem should be divided to component which will imports and realized with map()

Reply:
Then that means I need to create an array of strings, containing the text. However this must be corrected later when we implement translation in multiple languages. Do I proceed with the array idea?